packages/web-app/.storybook/en.json

```diff
@@ -1010,8 +1012,13 @@
   "theme": "theme",
   "Theme": "Theme",
   "There is no caves related to this group currently": "There is no caves related to this group currently.",
+  "There is currently no location for this entrance": "There is currently no location for this entrance.",
+  "There is currently no snapshot registered for this location": "There is currently no snapshot registered for this location.",
   "Therefore, for a few weeks, you will have the opportunity to respond to our <fundRaisingOperationUrl>fund-raising operation</fundRaisingOperationUrl>.": "Therefore, for a few weeks, you will have the opportunity to respond to our <fundRaisingOperationUrl>fund-raising operation</fundRaisingOperationUrl>.",
   "This {0} version of the site is improving gradually to allow a simpler navigation from all your peripherals": "This {0} version of the site is improving gradually to allow a simpler navigation from all your peripherals.",
+  "This entrance has a restricted access, you can not see its locations": "This entrance has a restricted access, you can not see its locations.",
+  "This location have no related snapshot": "This location have no related snapshot.",
+  "This location is related to a senstive entrance": "This location is related to a senstive entrance. Only administrator can see this.",
```

Comment:
This location is related to a senstive entrance -> This location is related to a sensitive entrance